### DISCUSSION TITLE
Is there a way of making the the modeline not flicker?

### DISCUSSION THREAD

**Discussion by joelkuiper** [joaotavora/eglot#1283] [General]
So I really like eglot except for one tiny thing that is honestly preventing me from enabling it right now. Whenever I type or edit something the modeline will briefly flash `/2` or whatever number after eglot:project-name (I'm using clojure-lsp for Clojure). Is there a way to prevent that from happening. I was unable to diminish the eglot modeline thing completely 

Anyway thanks in advance :sweat_smile:

**Comment by joelkuiper** [comment]
For reference this is what I ended up, which is crude but effective. 

```elisp
(setq mode-line-misc-info
        (delete '(eglot--managed-mode (" [" eglot--mode-line-format "] "))
                mode-line-misc-info))
```

**Reply by joaotavora** [reply]
I've been thinking of better ways to make this happen, as you probably just want to remove the ever-changing number of pending unanswered requests to the server from the mode-line.

But it takes some work, so your hack will have to do for now.
